src/Microsoft.AspNetCore.Identity/IdentityOptions.cs

```diff
@@ -11,6 +13,18 @@ namespace Microsoft.AspNetCore.Builder
     /// </summary>
     public class IdentityOptions
     {
+        public IdentityOptions(IEnumerable<IConfigureOptions<IdentityOptions>> configureOptions = null)
```

Comment:
yeah ... sorry ... I pulled that question b/c this was the wrong place to pose the question, and I read back in the main issue that you showed ...
```c#
// Just registers the MyOptions singleton
services.AddOptions<MyOptions>()

// Register the options singleton and configure it
services.AddOptions<MyOptions>(options => options.Setting = "Whatever"); //
```
... so I'm good. thx.

When it gets to the point of an Annoucement, if there are two ways (i.e., "If you want to avoid using AddOptions" approach), then I hope you'll just add in the pros and cons of each approach.